### PR TITLE
x86_cpu_flags: new case to test flag avx512_bf16

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 1861283

depends on [avocado-framework/avocado-vt#2672](https://github.com/avocado-framework/avocado-vt/pull/2672)

Signed-off-by: Nana Liu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
(1/1) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.intel.avx512_series_flags_5.q35: PASS (63.37 s)

**Comment by nanliu-r** [comment]
@huangyum @vivianQizhu Hi, please help to review this, thanks a lot.

**Comment by huangyum** [comment]
Seems this case has nothing to do with  avocado-framework/avocado-vt#2672 , should not depend on that.

File: qemu/tests/cfg/x86_cpu_flags.cfg [hunk 1]
```diff
@@ -14,3 +14,6 @@
                 - memory_protection_key:
                     cpu_model_flags += ",+pku,check"
                     flags = "pku ospke"
+                - avx512_series_flags_5:
+                    cpu_model_flags += ",avx512-bf16"
```
Comment (huangyum) [inline qemu/tests/cfg/x86_cpu_flags.cfg:18]: Missing '+' ?

Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_flags.cfg]: update and thanks.

File: qemu/tests/cfg/x86_cpu_flags.cfg [hunk 2]
```diff
@@ -14,3 +14,6 @@
                 - memory_protection_key:
                     cpu_model_flags += ",+pku,check"
                     flags = "pku ospke"
+                - avx512_series_flags_5:
+                    cpu_model_flags += ",+avx512-bf16"
+                    flags = "avx512_bf16"
```
Comment (huangyum) [inline qemu/tests/cfg/x86_cpu_flags.cfg:19]: Do you need restriction on guest? Does rhel7 guest work?

Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_flags.cfg:20]: Only rhel8.2.0 or later work and have updated, please help check.

 (1/2) Host_RHEL.m8.u3.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.intel.avx512_series_flags_5: CANCEL: The guest kernel doesn't support avx512_bf16 (2.20 s)
 (2/2) Host_RHEL.m8.u3.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.intel.avx512_series_flags_5: PASS (65.30 s)

Comment (vivianQizhu) [reply qemu/tests/cfg/x86_cpu_flags.cfg:20]: Why not `no RHEL.6, RHEL.7, RHEL.8.1`?

Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_flags.cfg:21]: > Why not `no RHEL.6, RHEL.7, RHEL.8.1`?

looks good, thanks and update.

Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_flags.cfg:21]: python3 ConfigTest.py --guestname=RHEL.8.0.0,RHEL.8.1.0,RHEL.8.2.0 --testcase=x86_cpu_flags.intel.avx512_series_flags_5 --clone=no

 (1/1) Host_RHEL.m8.u3.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.intel.avx512_series_flags_5: PASS (65.78 s)

**Comment by nanliu-r** [comment]
Host_RHEL.m8.u3.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.x86_cpu_flags.intel.avx512_series_flags_5: PASS (172.88 s)

**Comment by huangyum** [comment]
Please update commit summary as "x86_cpu_flags:...". Others look good to me.

**Comment by nanliu-r** [comment]
> Please update commit summary as "x86_cpu_flags:...". Others look good to me.

update and thanks

**Pull request merged**